### ISSUE TITLE
SideDrawer issue

### ISSUE THREAD

**Issue by stevenleeza** [NativeScript/nativescript-marketplace-demo#226]
Tap (quickly) on the left edge of the screen to get the SideDrawer to open slightly (a few pixels) then close again.

Now try to tap on any UI component - nothing happens.

Problem: The SideDrawer, although closed, still has an invisible layer over the entire screen area so nothing can be tapped. You can get everything back to normal by swipe opening the SideDrawer completely and then tapping/swiping it back to close it.

This issue can be replicated on any app that makes use of the latest Telerik UI SideDrawer component.

**Comment by tsonevn** [comment]
This has  been fixed with [nativescript-telerik-ui version 1.4.0](https://www.npmjs.com/package/nativescript-telerik-ui)

**Comment by stevenleeza** [comment]
Thanks Nikolay for the update.
I've updated to the latest version but the problem still persists ("master" branch).
The issue is present on Android (I haven't tested iOS yet).
I guess I'll have to submit a screen recording so everyone can see what I mean.

**Comment by tsonevn** [comment]
Hi @stevenleeza.
Thank you for your reply, 
I tested the given case again and confirmed that this is a real issue with the SideDrawer. However I also found that you have opened an issue about this behavior [here](https://github.com/telerik/nativescript-ui-samples/issues/125) in `nativescript-telerik-ui` repo. In my opinion this issue is not related  with this app. The app uses latest official  for `nativescript-telerk-ui`. When `nativescript-telerk-ui` provides a fix for this behavior, it will be available and for this application too. 

I hope this is information helps.
